Swiftification/Array.swift

```diff
@@ -29,4 +29,35 @@ public extension Array {
         return (indices ~= index ? self[index] : nil)
     }
     
+    /// Index of the first occurrence of item, if found.
+    func indexOf <U: Equatable> (item: U) -> Int? {
+        if item is Element {
+            return self.indexOf({ (object) -> Bool in
+                return (object as! U) == item
+            })
+        }
+        
+        return nil
+    }
+    
+    /// Returns the first occurence of item, if found
+    func find <U: Equatable> (item: U) -> Element? {
```

Comment:
That's what I get for stealing. I was going for a find that takes a closure and returns the first occurrence where the closure returns true